core-plugins/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -74,7 +74,7 @@ overrideSchema=[{"key":"id","value":"float"},{"key":"firstname","value":"string"
   {"key":"lastname","value":"string"},{"key":"photo","value":"string"},{"key":"outputfield","value":"string"}]
 invalidOverrideField=invalid
 overrideFields=[{"key":"id","value":"float"}]
-readRecursivePath=testdata/file/RECURSIVE_TEST
+readRecursivePath=gs://testdata-core/RECURSIVE_TEST
```

Comment:
This will break if we run tests in a project that doesn't have permissions to access bucket `testdata-core`. It would be better to store input files in this repo and copy them to a temp GCS bucket (created for each test run, deleted at the end) which is used in the test

Reply:
Sure, added a test setup hook to create and delete a GCS bucket for this Folder. (181bad4e22257f41713fc27864db524b85146da0)

Reply:
Also the files which we are using in data validation are also stored in GCS bucket testdata-core. So I believe we should follow the above approach for those files as well and create temp buckets for them.